### PR TITLE
chore: update Graphcast SDK and fix conflicts

### PR DESCRIPTION
**Pull request by cargopete**
Updated Graphcast SDK to 0.6.0 and fixed conflicts 🖖🏻

### CODE REVIEW

File: src/operator/radio_types.rs [hunk 1]
```diff
@@ -10,7 +13,7 @@ use serde::{Deserialize, Serialize};
 // EIP-712 name will change encoding, append other unique message types/names to decode properly
 #[derive(Eip712, EthAbiType, Clone, Message, Serialize, Deserialize, PartialEq, SimpleObject)]
 #[eip712(
-    name = "Graphcast POI Radio",
+    name = "Graphcast Listener Radio",
```
Comment (hopeyen) [inline src/operator/radio_types.rs]: iirc EIP712 name for message types must stay consistent between radios, otherwise they won't parse correctly

Comment (cargopete) [reply src/operator/radio_types.rs]: Made it consistent with the naming of other message types, lmk what you think

**Review by hopeyen — DISMISSED** [review]
:+1:

**Review by hopeyen — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**